.github/workflows/validate.yaml

```diff
@@ -2,16 +2,40 @@ name: Validate
 
 on:
   push:
-  pull_request:
```

Comment:
This is not needed, because we already do it on every push; so PRs will be checked anyway.